code/modules/fish/fish_items.dm

```diff
@@ -152,15 +152,15 @@
 	name = "shark teeth"
 	desc = "A number of teeth, supposedly from a shark."
 	icon = 'icons/obj/fish_items.dmi'
-	icon_state = "teeth"
 	force = 2.0
 	throwforce = 5.0
 	materials = list()
 
 /obj/item/shard/shark_teeth/Initialize(mapload)
 	. = ..()
-	src.pixel_x = rand(-5,5)
-	src.pixel_y = rand(-5,5)
+	icon_state = "teeth"
```

Comment:
Then you can just add a comment explaining why icon_state needs to be reset

Reply:
yeah, will do.